source/Main.cpp

```diff
@@ -40,11 +42,43 @@ int main(int argc, char* argv[]) {
   tool.add_options(options);
 
   try {
+    if (argc < 2) {
+      std::cerr << "Usage: " << argv[0] << " <json_config_file>\n";
+      return ExitCode::invalid_argument_error("No JSON configuration file provided.");
+    }
```

Comment:
We should use boost program options to get the file name, instead of parsing the argument ourselves.
See https://www.boost.org/doc/libs/1_58_0/doc/html/program_options/tutorial.html#idp337609360
We need to use `positional_options_description`